test/e2e/app-dir/with-exported-function-config/with-exported-function-config.test.ts

```diff
@@ -11,34 +11,38 @@ describe('with-exported-function-config', () => {
         await next.readFile('.next/server/functions-config-manifest.json')
       )
 
-      expect(functionsConfigManifest).toMatchInlineSnapshot(`
-        {
-          "functions": {
-            "/api/page-route": {
-              "maxDuration": 1,
-            },
-            "/app-route": {
-              "maxDuration": 1,
-            },
-            "/app-route-edge": {
-              "maxDuration": 2,
-            },
-            "/app-ssr": {
-              "maxDuration": 3,
-            },
-            "/app-ssr-edge": {
-              "maxDuration": 4,
-            },
-            "/page": {
-              "maxDuration": 2,
-            },
-            "/page-ssr": {
-              "maxDuration": 3,
-            },
-          },
-          "version": 1,
-        }
-      `)
+      expect(functionsConfigManifest).toEqual({
```

Comment:
Why change this? Seems fine to use the snapshot here which makes it easier to update if things change.

Reply:
I thought it easier to manage as we can use object properties and also add comments to specific like 522b9537d5d249e39e606f4f69459ac9f9519ded.